Sources/_OpenAPIGeneratorCore/Translator/Content/ContentType.swift

```diff
@@ -143,21 +182,25 @@ struct ContentType: Hashable {
         "\(lowercasedType)/\(lowercasedSubtype)"
     }
 
-    /// Returns the type and subtype as a "<type>\/<subtype>" string.
+    /// Returns the type and subtype as a "<type>\/<subtype>[;<param>...]" string.
```

Comment:
Small nit: maybe worth updating to something like this?
```suggestion
    /// Returns the type, subtype and parameters (if present) as a "<type>\/<subtype>[;<param>...]" string.
```